apm-sniffer/apm-sdk-plugin/sharding-jdbc-1.5.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/sjdbc/ExecuteEventListener.java

```diff
@@ -83,6 +83,7 @@ private void handle(AbstractExecutionEvent event, String operation) {
                 if (event.getException().isPresent()) {
                     span.log(event.getException().get());
                 }
+                // fall through
```

Comment:
Is this meaningful for the test tool?

Reply:
![image](https://user-images.githubusercontent.com/26432832/94820216-c8487600-0432-11eb-8c2b-19358e936dea.png)

Yes

Reply:
OK, interesting.